.golangci.yml

```diff
@@ -25,7 +20,28 @@ linters:
     - unconvert
     - unused
     - whitespace
-
-linters-settings:
-  goimports:
-    local-prefixes: sigs.k8s.io/metrics-server
+  exclusions:
+    generated: lax
+    presets:
+      - comments
+      - common-false-positives
+      - legacy
+      - std-error-handling
+    paths:
+      - third_party$
+      - builtin$
+      - examples$
+formatters:
+  enable:
+    - gofmt
+    - goimports
+  settings:
+    goimports:
+      local-prefixes:
+        - sigs.k8s.io/metrics-server
+  exclusions:
+    generated: lax
+    paths:
+      - third_party$
+      - builtin$
+      - examples$
```

Comment:
This file was updated running the command `golangci-lint migrate
` as mentioned in https://golangci-lint.run/product/migration-guide/